pkg/apis/crd/v1alpha2/types.go

```diff
@@ -202,6 +201,15 @@ type Egress struct {
 
 	// Specification of the desired behavior of Egress.
 	Spec EgressSpec `json:"spec"`
+
+	// EgressStatus represents the current status of an Egress.
+	Status EgressStatus `json:"status"`
+}
+
+// EgressStatus represents the current status of an Egress.
+type EgressStatus struct {
+	// The name of the Node that holds the Egress IP.
+	NodeName string `json:"nodeName"`
```

Comment:
Probably make it clearer the Node holds the Egress IP like "EgressNode"?

Reply:
NodeName-->EgressNode?

Reply:
I think yes

Reply:
done